src/core/api/api_error.pl

```diff
@@ -1087,6 +1087,13 @@
                               'api:key' : Key},
              'api:message' : Msg
             }.
+api_error_jsonld_(diff,error(table_diff_timeout,_), JSON) :-
+    format(string(Msg), "The table diff you requested is too complex for our little brains. Perhaps try signing up for a dedicated table diff account?", []),
+    JSON = _{'@type' : 'api:DiffErrorResponse',
+             'api:status' : 'api:i_am_a_teapot',
```

Comment:
:D But really, we shouldn't be using this, as terminusdb is in fact not a teapot. Considering this message is trying to push people onto a paid tier, maybe 402 payment required? Apparently there's precedent - wikipedia says google uses it when people exceed limits on their APIs.

Reply:
payment required it is!